ui/raidboss/data/02-arr/raid/t8.ts

```diff
@@ -112,18 +116,19 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
+      netType: 'StartsUsing',
```

Comment:
> I had to additionally discriminate between NetRegexTrigger and RegexTrigger, so the former has netType and the latter has type, to match the naming schema. type never needs to be specified, it just has to exist for typescript to properly discriminate between NetRegex and Regex.

Forgive me for being skeptical, but how does an optional field contribute to discrimination?

Things seem to work if I change `PartialRegexTrigger` to be defined like this (and `s/PartialRegexTrigger<'None'>/PartialRegexTrigger/`):
```typescript
type PartialRegexTrigger = {
  regex: RegExp;
  regexDe?: RegExp;
  regexFr?: RegExp;
  regexJa?: RegExp;
  regexCn?: RegExp;
  regexKo?: RegExp;
};
```

Mostly I say this, because if I had my druthers, I would much rather just call this `type` everywhere and not bother with type inference on `RegexTrigger`.

My other feedback is that I also would order `type` after id but before `netRegex`.  I think `id` being first is helpful for scanning trigger files.

Could you also update the following to mention `type` and its ordering?
* [this lint](https://github.com/quisquous/cactbot/blob/main/eslint/cactbot-trigger-property-order.js#L29)
* [this test](https://github.com/quisquous/cactbot/blob/main/test/helper/test_trigger.js#L329)
* [this doc](https://github.com/quisquous/cactbot/blob/main/docs/RaidbossGuide.md#trigger-structure)
* [responses](https://github.com/quisquous/cactbot/blob/main/resources/responses.ts#L37)

Reply:
I tested this out a bit more this morning, and it does seem to work fine with `type` and no second discriminator. I'm really not sure why it wasn't working last night, maybe just leftover issues from when eslint/tsc broke on the more-than-50-union-members issue.